tests/integration/cos_substrate.py

```diff
@@ -241,24 +242,24 @@ def execute_command(self, container, command: List[str]):
         Returns:
             Tuple[int, bytes, bytes]: rc, stdout, and stderr
         """
-        log.info("Running command")
+        _cmd = shlex.join(command)
+        log.info("Running command: %s", _cmd)
         try:
-            rc, stdout, stderr = container.execute(command)
+            rc, stdout, stderr = container.execute(command, decode=False)
             if rc != 0:
                 log.error(
                     "Failed to run %s with return code %s. stdout: %s, stderr: %s",
-                    command,
+                    _cmd,
                     rc,
                     stdout,
                     stderr,
                 )
```

Comment:
In this case, we have to decode the stdout and stderr, right?

Reply:
by using `decode=False` -- it forces the output/errors to bytes.  Without this -- we were floating based on what the library set as the default.  It turns out it changed (and we were depending on `decode=True`) for everything to work